app/src/androidTest/java/org/mozilla/fenix/ui/screenshots/ThreeDotMenuScreenShotTest.kt

```diff
@@ -78,49 +77,20 @@ class ThreeDotMenuScreenShotTest : ScreenshotTest() {
         }.openThreeDotMenu {
         }
         settingsButton2()
+        onView(withId(R.id.recycler_view)).perform(swipeUp())
```

Comment:
These two lines might better be put in the `Robot` for this screen.

Reply:
> These two lines might better be put in the `Robot` for this screen.

I spoke with @isabelrios this morning.  She's going to go ahead and land this for now since she's waiting for updates on the robot to stick this in.